src/includes/navigation-services.njk

```diff
@@ -147,14 +149,11 @@
                 <a class="nav-link ps-2 d-flex justify-content-between cursor-pointer align-items-center" href="/team/">Team</a>
                 </li>
                 <li class="nav-item mx-2">
-                <a class="nav-link ps-2 d-flex justify-content-between cursor-pointer align-items-center" href="/pages/case-study">Case Study</a>
+                <a class="nav-link ps-2 d-flex justify-content-between cursor-pointer align-items-center" href="/about-us/">Playlists</a>
```

Comment:
About us > Playlists? 
Should we try to make this stuff auto generated off of collections?